wiki/Client/Beatmap_editor/Song_setup/en.md

```diff
@@ -1,238 +1,121 @@
 
-### Countdown
-
-Check the `Enable countdown` checkbox to enable the countdown animation cheer before the beatmap starts.
-
-When enabled, the countdown animation can be refined using the `Countdown Speed` radio buttons (`Half`, `Normal`, `Double`) and the `Countdown Offset` (default is 0; higher value means earlier countdown).
-
-### Misc. Toggles
-
-- `Widescreen Support`: Remove [sidebars/pillarboxes](https://en.wikipedia.org/wiki/Pillarbox) from the left and right side of the playfield if the client is using any aspect ratio greater than `4:3`. This is typically enabled when using storyboard elements but can be disabled.
-- `Display storyboard in front of combo fire`: An obsolete setting, this can be ignored.
-- `Display epilepsy warning (storyboard has quick strobing)`: Display an epilepsy warning for (quick or any) flashing storyboards in the beatmap.
-- `Letterbox during breaks`: Determine if the beatmap should (or should not) put [letterboxes](https://en.wikipedia.org/wiki/Letterboxing_(filming)) during breaks in the beatmap. This is usually enabled by default, but using this option in for osu!mania-specific beatmaps is not allowed.
+`Design` contains various settings which affect the beatmap's overall look and feel.
 
-### Skinning
-
-![Screenshot of the Skinning tab](img/SS_Skin_eg.jpg "Skinning tab")
-
-The `Preferred Skin` dropdown is used to set the preferred skin for the beatmap. Using this option assumes that the player has the skin you want them to use. However, if the player does not have the skin, a notification will appear for the player, stating that the requested skin is missing, but the player's current skin will be used instead. By default, `User's preference (No Change)` is used.
-
-You can, however, include the skin elements inside the beatmap folder. As most creators will just do this instead, this option is rarely used unless it would decrease the file size or is part of a beatmap project, where the skin may be provided on the project page itself.
+| Control | Action |
+| :-- | :-- |
+| `Enable countdown` | Activate the [countdown animation](/wiki/Beatmap/Countdown) before the beatmap starts. |
+| `Countdown Speed` | Customize the countdown speed. `Half`: the countdown takes 8 [full beats](/wiki/Music_theory/Beat). `Normal`: 4 beats. `Double`: 2 beats. |
+| `Countdown Offset` | How many beats earlier the countdown should start. |
+| `Widescreen Support` | Remove [pillarboxes](https://en.wikipedia.org/wiki/Pillarbox) from the sides of the playfield if the screen's aspect ratio is higher than `4:3`. This is typically enabled when using widescreen storyboard elements. |
+| `Display storyboard in front of combo fire` | Put the [storyboard](/wiki/Storyboard) before the [combo fire](/wiki/Gameplay/Combo_fire). *Note: this setting has become obsolete with the removal of the combo fire.* |
+| `Display epilepsy warning` | Before the start of the beatmap, display the epilepsy warning in case of quickly flashing storyboards. |
```

Comment:
```suggestion
| `Display epilepsy warning` | Before the start of the beatmap, display an epilepsy warning in case of quick strobing in the video or storyboard. |
```